src/bootstrap/src/core/builder/cli_paths/snapshots/x_install.snap

```diff
@@ -2,15 +2,9 @@
 source: src/bootstrap/src/core/builder/cli_paths/tests.rs
 expression: install
 ---
-[Install] install::Docs
-    targets: [aarch64-unknown-linux-gnu]
-    - Set({install::src/doc})
-[Install] install::Std
```

Comment:
hm, where did these go? I see we have our own install::Std below, but I still see the upstream one marked `is_default() { true }`, and `install:Docs` and `install::LlvmTools` seem to have disappeared altogether.